dbt/adapters/athena/impl.py

```diff
@@ -95,14 +95,14 @@ class AthenaConfig(AdapterConfig):
     partitioned_by: Optional[str] = None
     bucketed_by: Optional[str] = None
     bucket_count: Optional[str] = None
-    table_type: Optional[str] = "hive"
-    ha: Optional[bool] = False
-    format: Optional[str] = "parquet"
+    table_type: str = "hive"
+    ha: bool = False
+    format: str = "parquet"
     write_compression: Optional[str] = None
     field_delimiter: Optional[str] = None
     table_properties: Optional[str] = None
     native_drop: Optional[str] = None
-    seed_by_insert: Optional[bool] = False
+    seed_by_insert: bool = False
```

Comment:
isn't this Optional?

Reply:
Since there is a default we don't need it I think

Reply:
Either we put None as a default value or we remove the Optional

Reply:
https://github.com/dbt-athena/dbt-athena/pull/441#discussion_r1350125783